gradle-plugin/src/main/groovy/akka/grpc/gradle/AkkaGrpcPlugin.groovy

```diff
@@ -140,7 +141,9 @@ class AkkaGrpcPlugin implements Plugin<Project>, DependencyResolutionListener {
     @Override
     void beforeResolve(ResolvableDependencies resolvableDependencies) {
         def compileDeps = project.getConfigurations().getByName("compile").getDependencies()
-        compileDeps.add(project.getDependencies().create("com.lightbend.akka.grpc:akka-grpc-runtime_2.12:${pluginVersion}"))
+        boolean foundRuntimeDep = compileDeps.any { dep -> dep.group == "com.lightbend.akka.grpc" && dep.group.startsWith("akka-grpc-runtime") && dep.version == pluginVersion }
+        if(!foundRuntimeDep)
+            throw new RuntimeException("""Missing dependency:   implementation 'com.lightbend.akka.grpc:akka-grpc-runtime_\${scalaVersion}:${pluginVersion}' """)
```

Comment:
I've tested the removal of the dependency but I haven't tested the verification + exception.